crypto/types.go

```diff
@@ -0,0 +1,9 @@
+package crypto
+
+import tmcrypto "github.com/tendermint/tendermint/crypto"
+
+type (
+	PubKey  = tmcrypto.PubKey
+	PrivKey = tmcrypto.PrivKey
+	Address = tmcrypto.Address
```

Comment:
Do we really need this aliasing thing here? How about just dropping these aliases?

Reply:
The idea is to eventually have our own `PubKey` interface (discussed in #7008). Maybe I should do that in a separate PR, but aliasing is a first step. I can remove here